Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Slot.cpp

```diff
@@ -958,6 +959,13 @@ namespace ScriptCanvas
         return m_node->ConstructTransientIdentifier((*this));
     }
 
+    void Slot::OnVariableRenamed(AZStd::string_view variableName)
+    {
+        DisconnectVariableNotificationBus();
+
```

Comment:
I don't understand the reasoning behind disconnecting for the rename, and also, aren't the event supposed to be re-connected after the action ?

Reply:
Good catch, it wasn't necessary, leftover from debugging/diagnosing